lms/djangoapps/discussion/templates/discussion/edx_ace/response_notification/email/head.html

```diff
@@ -0,0 +1,29 @@
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title>{% block title %}Reply to {{ thread_title }} at {{ comment_created_at }} {% endblock %}</title>
```

Comment:
We'll have to move this whole header out into a helper for analytics purposes, but I can do that in my PR.